ktlint-core/build.gradle

```diff
@@ -9,8 +9,16 @@ dependencies {
   api deps.logging
   api deps.logback
 
-  testImplementation deps.junit
+  // Standard ruleset is required for EditConfigLoaderTest only
+  testImplementation project(":ktlint-ruleset-standard")
+
+  testImplementation(deps.junit5) {
+    because 'allows to write and run Jupiter tests'
+  }
   testImplementation deps.assertj
```

Comment:
No need to declare these `because`? It's obvious.
https://github.com/Goooler/ktlint/commit/d34f8ca45c0231ff5cb79b26932cf4e1db1469fb

Reply:
Yeah, it is trivial and needless. Just copy-pasta without understanding stuff. Is now cleaned up.

Reply:
Missed to push?

Reply:
Indeed. Tnx for notifying.